src/Geta.Optimizely.Sitemaps/Services/NullUriAugmenterService.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using EPiServer.Core;
+using EPiServer.ServiceLocation;
+
+namespace Geta.Optimizely.Sitemaps.Services
+{
+    [ServiceConfiguration(typeof(IUriAugmenterService))]
+    public class NullUriAugmenterService : IUriAugmenterService
```

Comment:
I think the name `NullUriAugmenterService` is misleading - `Null` services usually do not do anything (returns nothing), but this one returns the URL provided.

Better call it `DefaultAugmenterService` - it would mean that the service has some default behavior.